bower.json

```diff
@@ -17,6 +17,10 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^3.0.0"
+    "purescript-prelude": "^3.0.0",
+    "purescript-foldable-traversable": "^3.3.0"
```

Comment:
One option is to put a `traverseRec`/`traverseRec_` into `Foldable`/`Traversable`, which would use `MonadRec`, and then to implement it for `Array`, `List`, etc., but that seems unpleasant for a bunch of reasons.

I'm going to think about how we might be able to create something general for consuming things one at a time without creating garbage.